Amplify/Categories/Auth/Request/AuthConfirmSignUpRequest.swift

```diff
@@ -25,17 +25,14 @@ public struct AuthConfirmSignUpRequest: AmplifyOperationRequest {
 public extension AuthConfirmSignUpRequest {
 
     struct Options {
-        //TODO: Add documetnation.
-        public let forceAliasCreation: Bool
-        public let validationData: [String: String]?
-        public let metadata: [String: String]?
-
-        public init(forceAliasCreation: Bool = false,
-                    validationData: [String: String]? = nil,
-                    metadata: [String: String]? = nil) {
-            self.forceAliasCreation = forceAliasCreation
-            self.validationData = validationData
-            self.metadata = metadata
+
+        /// Extra plugin specific options, only used in special circumstances when the existing options do not provide
+        /// a way to utilize the underlying auth plugin functionality. See plugin documentation for expected
+        /// key/values
+        public let pluginOptions: Any?
```

Comment:
This wording is confusing, but maybe I'm just tired. Is this how we word other documentation of `pluginOptions` in other request options?